oasislmf/utils/data.py

```diff
@@ -409,13 +415,14 @@ def get_dataframe(
                 # Find flexible fields in loc file and set their data types to that of
                 # FlexiLocZZZ
                 if 'FlexiLocZZZ' in col_dtypes.keys():
-                    headers = list(pd.read_csv(src_fp, encoding=use_encoding).head(0))
+                    headers = list(pd.read_csv(src_fp, encoding=use_encoding).head(0), low_memory=low_memory)
```

Comment:
I guess you saw in the error but you put the arg in the wrong place
headers = list(pd.read_csv(src_fp, encoding=use_encoding, low_memory=low_memory).head(0))

Reply:
oupf! thanks for the spot